src/main/java/io/cdap/e2e/utils/ConstantsUtil.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.utils;
+
+/**
+ * Collection of Constants
+ */
+public class ConstantsUtil {
+  public static final String AUTOMATION_TEST = "automation_test";
+  public static final String PROJECT_ID = "Project-ID";
+  public static final String DATA_SET = "Data-Set";
```

Comment:
change the value of "PROJECT_ID" and value to "projectId"
change the variable name to "DATASET" and value to "dataset".
And accordingly modify at the places where it is getting used.

Reply:
Hi @itsankit-google , 
If you are asking me to change 'PROJECT_ID to PROJECTID', I don't see anything wrong with the variable name, they are perfectly as per naming convention. Please confirm if you still want me to change them.
But for properties keys, we can make it to all small for uniform visibility. This one i will address

Reply:
@sumitnigamcs, I think there was some misunderstanding to be clear, I meant line no 23 to be like this,
`public static final String PROJECT_ID = "projectId";`
and line no 24 to be like this,
`public static final String DATASET = "dataset";`

Reply:
ok got it